### ISSUE TITLE
[CLI] - CopyWebpackPlugin - ignore options not working

### ISSUE THREAD

**Issue by m0jimo** [quasarframework/quasar#7433]
When I was a looking for a way how to add custom folder to be ignored when building the app with `quasar build` I discovered that Quasar code handling icons for SPA and Electron doesn't work ([Discord chat](https://discord.com/channels/415874313728688138/601046346602119168/732903815564623872)).

It shouldn't copy icons when ['electron', 'cordova', 'capacitor'] is used. [link to Quasar DEV CLI code- CopyWebpackPlugin - ignore](https://github.com/quasarframework/quasar/blob/6805244a67e539e2b5e1ff1cde74e8db3849e689/app/lib/webpack/create-chain.js#L322)
```
const patterns = [{
        from: appPaths.resolve.app('public'),
        to: '.',
        noErrorOnMissing: true,
        globOptions: {
          ignore: [ appPaths.resolve.app('/**/.*') ].concat(
            // avoid useless files to be copied
            ['electron', 'cordova', 'capacitor'].includes(cfg.ctx.modeName)
              ? [ appPaths.resolve.app('public/icons'), appPaths.resolve.app('public/favicon.ico') ]
              : []
          )
        }
      }]
```
### Result for Electron
```
 Build mode........ electron
 Pkg quasar........ v1.12.11
 Pkg @quasar/app... v2.0.5
 Debugging......... no
 Publishing........ no
```
![image](https://user-images.githubusercontent.com/37825539/87633451-b90f3180-c73b-11ea-947d-e0d3a1d6703f.png)

### Result for SPA
```
 Build mode........ spa
 Pkg quasar........ v1.12.11
 Pkg @quasar/app... v2.0.5
 Debugging......... no
 Publishing........ no
```
![image](https://user-images.githubusercontent.com/37825539/87633502-c7f5e400-c73b-11ea-95d1-e9b88dbd1a78.png)

Problem is in path definition (inspiration from [copy-webpack-plugin issue](https://github.com/webpack-contrib/copy-webpack-plugin/issues/498)):
`[ appPaths.resolve.app('public/icons'), appPaths.resolve.app('public/favicon.ico') ]` should be probably changed to ` ['**/public/icons', '**/public/favicon.ico']` - after changes it copy the folders only for SPA.
```
 const patterns = [{
        from: appPaths.resolve.app('public'),
        to: '.',
        noErrorOnMissing: true,
        globOptions: {
          ignore: ['/**/.*'].concat(
            // avoid useless files to be copied
            ['electron', 'cordova', 'capacitor'].includes(cfg.ctx.modeName)
              ? ['**/public/icons', '**/public/favicon.ico']
              : []
          )
        }
      }]
```
And the result:
![image](https://user-images.githubusercontent.com/37825539/87634270-3a1af880-c73d-11ea-8093-796c50d053d9.png)

I'll be happy If you could provide an example how to correctly inject my custom ignore folder in `quasar.conf.js` instead of hacking source `create-chain.js` file.

**Comment by webnoob** [comment]
Will provide an update to fix the ignore stuff later today.

RE Quasar conf:

Under build:

```
chainWebpack (chain) {
  chain.plugins.store.get('copy-webpack').store.get('args')[0].patterns[0].globOptions.ignore.push('**/donotcopy/**/*')
}
```

**Comment by webnoob** [comment]
Available in quasar/app-v2.0.6
